src/test/java/com/wikia/webdriver/elements/mercury/old/JoinPageObject.java

```diff
@@ -26,27 +24,25 @@
 
   private WebDriver driver;
   private Wait wait;
-  private UrlBuilder urlBuilder;
+  private UrlBuilder urlBuilder = UrlBuilder.createUrlBuilder();;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this empty statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AEmptyStatementUsageCheck)